st-embroider/2023/2003-01-23.md

```diff
@@ -0,0 +1,130 @@
+  - sorry I was talking and couldn't take notes 🙈
+  - rough conclusion: the issue is likely in build-compat-addons.ts and related to "smooshing"
+  - there is a part that is outputing an empty package, this is a good time to drop a debugger and watching the code work
+  - EF: this is all likely going to go away when we move to a resolver implementation of embroider (instead of the heavy-handed copy whole tree implementation)
+  - 
+<10-15min> Unblock v2 conversion of ember-responsive-image: <@simon>
+https://github.com/embroider-build/embroider/issues/1305 
+  - opt into new format for index.html with a flag
+  - a bit vague and I need to give more details
+https://github.com/ef4/ember-auto-import/issues/554 (Importing assets with query params does not work)
+  - it's likely the embroider-auto-import splitter that needs a change
+  - the define is being driven by data that is coming out of the splitter
+  - you will likely need to add the correct implementation to the `webpack.ts` entryTemplate
+  - in the issue there is a missing define, the define example in the issue is showing it as a consumer so something is going to need to define what 
+  - (discussion about other odd things in the build - a bit of investigation)
+  - slightly trolly thing: you need to add webpack config to autoImport **and** the embroider config in your ember-auto-import
+  - if we want to allow people to import things from their own app in ember-auto-import then we will need to implement the first feature we discussed today
+  - re: trolly webpack config there are a few options that we can do. open to suggestions to make it better
+<5m> should ember-auto-import be under the embroider org? (related to release conversation and bus number) <@mansona>
+  - maybe
+  - CM: this is about allowing more peopel to release the important things
+  - EF: sure sounds good
+  - EF: we could move it to the embroider monorepo
+  - CM: I wasn't suggesting that 🙈
+<2m> ember-cli-babel v8 https://github.com/embroider-build/embroider/pull/1334 <@nullvoxpopuli>
+  - (discussion about the new semantics of ember-cli-babel)
+  - consensus: let's just merge this
+  - (more discussion)
+  - CM: let's comment the changes and we can get this in the next release. Releasing is more urgent
+  - EF: sure
+
+Do a release
+  - CM: can we record it
+  - EF: probably written is better
+  - CM: I will do that with the notes
+
+
+  process: 
+    - I look at the changelog and find the release name
+    - find the corresponding tag
+    - I look at the git log and search for "Merge pull" and make sure that all pull requests are covered
+    - (trying to remember changesets)
+   
+    - asside there is a lot of manual work
+    - NVP: that's because we forgot to add changeset to PRS before merging
+    - CM: can't make that the PR authors job, and can't put it on the maintainer
+    - EF: I need a list of all changes per package so we know what packages need to be released
```

Comment:
I opened an issue here to help: https://github.com/changesets/changesets/issues/1066